.github/workflows/website-validation.yml

```diff
@@ -4,6 +4,8 @@ on:
     types: [opened, synchronize, reopened]
     paths:
       - "docs/**"
+      - "*.md"
+      - "**/*.md"
```

Comment:
Is it possible to exclude, say, the server and build-tools folders?

Reply:
If it's not straightforward then ignore it.

Reply:
Updated to exclude the folders using `paths-ignore`